### PR TITLE
Emphasized suggestion to create separate geojson

### PR DESCRIPTION
**Pull request by thaisbendixen42**
Added a sentence that emphasizes the generation of separate geojson file. 

### Checklist

- [x] Checked spelling and grammar
- [x] Checked the output of `make html` to ensure new content displays correctly

### CODE REVIEW

File: source/reference/data-transfer.rst [hunk 1]
```diff
@@ -143,10 +143,9 @@ As mentioned in the :ref:`section on specifications
 with property keys that contain the values that correspond to those
 capabilities.
 
-In the case of imagery or large amounts of vector data (in formats
-such as GeoJSON, Shapefiles or KML), where this would become impractical
-to add to the JSON itself, the value should simply be a file path,
-**relative** to the main metadata file.
+In the case of imagery or vector data (in formats such as GeoJSON, Shapefiles
```
Comment (markusuwe) [inline source/reference/data-transfer.rst:146]: Paragraph before is not correct any more and we can probably get a stronger formulation:

As mentioned in the section on specifications above, blocks should output GeoJSON features with the property key "up42.data_path" pointing to the created output file(s). In the case of imagery or vector data (in formats such as GeoJSON, Shapefilesor KML), the value should simply be a file path, **relative** to the main metadata file.

**Review by markusuwe — COMMENTED** [review]
The paragraph needs a bit more rephrasing, I added a suggestion.

**Comment by chrieke** [comment]
@thaisbendixen42 Wanted to send you some suggestions but got too complicated. Not good practice but now just pushed to this PR, please feel free to revert to your former commit or mix it up. 

Removed the old capabilities section and some unneccessary examples. The chapter should be visited again at some point, some stuff on the bottom is still todo and it doesn't really stand on it's own, probably best to integrate it into some "how to build a block" main chapter.

File: source/reference/data-transfer.rst [hunk 2]
```diff
@@ -61,19 +59,15 @@ is an arbitrary key/value mapping.  A ``Feature`` can have an
...
-++++++++++++++++++++++++++++++++++++++
+up42.data_path property
++++++++++++++++++++++++
 
-To reduce the amount of guesswork required to pass data between
-blocks, every :ref:`capability <block-capabilities>` that a block
-outputs should have a corresponding key in the ``"properties"`` field
-of the features that it generates.
+Any feature in the ``data.json`` should contain a property ``up42.data_path`` with a relative file path
+pointing to the respective file in the ``tmp/output`` folder. This is the case for output imagery
```
Comment (marooned20) [inline source/reference/data-transfer.rst:68]: minor yet important
```suggestion
pointing to the respective file in the ``/tmp/output`` folder. This is the case for output imagery
```

File: source/reference/data-transfer.rst [hunk 3]
```diff
@@ -82,18 +76,36 @@ corresponding output should look something like this:
...
+                    // ...
+                }
+            },
+            {
+                "type": "Feature",
+                "geometry": {
+                    // ...
+                },
+                "properties": {
+                    "up42.data_path": "be051fa1/rgb.tif",
```
Comment (marooned20) [inline source/reference/data-transfer.rst:93]: In leu of the mentioned vector data file, I guess, as an example `block_output_vectors.geojson` would bring extra clarity and make it mistake-proof hopefully.

File: source/reference/data-transfer.rst (hunk 2 above)
Comment (markusuwe) [inline source/reference/data-transfer.rst:68]: This does not have to be a file, but can also be an output directory.

**Review by markusuwe — COMMENTED** [review]
LGTM

File: source/reference/data-transfer.rst (hunk 3 above)
Comment (rodrigoalmeida94) [reply source/reference/data-transfer.rst:93]: Second this!

**Review by perusio — APPROVED** [review]
LGTM. Thank you for looking into this @thaisbendixen42

**Review by perusio — APPROVED** [review]
LGTM.

**Review by teoEnache — APPROVED** [review]
LGTM

**Pull request merged**